File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-08_C.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    const a: i64 = 7;
    const b: i64 = 0;
    const q: i64 = @divTrunc(a, b);

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=QUOT:{d}\n", .{q}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-08_C.rs
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let a: i64 = 7;
    let b: i64 = 0;
    let q: i64 = a / b;

    writeln!(out, "OBS=QUOT:{}", q).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
